### PR TITLE
Edit header

### PR DESCRIPTION
**Pull request by maksim-luzin**

### CODE REVIEW

File: frontend/src/components/Header/index.tsx [hunk 1]
```diff
@@ -14,79 +23,317 @@ import LanguageSwitcher from '../SwitchLanguageOption/LanguageSwitcher';
...
+      setHideOnScroll(true);
+      setProfileMobileMenu(false);
+      setMobileMenu(false);
+      setProjectsMobileMenu(false);
+    } else {
+      setHideOnScroll(false);
+    }
+    prevScrollPos = currentScrollPos;
+  };
+
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Header/index.tsx]: Such logic should be inside useEffect and take a look on useScroll, please

File: frontend/src/components/Header/index.tsx [hunk 2]
```diff
@@ -14,79 +23,317 @@ import LanguageSwitcher from '../SwitchLanguageOption/LanguageSwitcher';
...
+              </div>
+            )}
+            placement="bottom-end"
+            id="id"
+            rootClose
+            className={classes.menu_user_container}
+          >
+            {() => (
+              <div className={classes.menu_user}>
+                <Link to={Routes.PROFILE}>View account</Link>
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Header/index.tsx]: Localization

File: frontend/src/components/Header/index.tsx [hunk 3]
```diff
@@ -14,79 +23,317 @@ import LanguageSwitcher from '../SwitchLanguageOption/LanguageSwitcher';
...
+              <div
+                className={classes.mobile_menu_projects}
+              >
+                <div
+                  className={`
+                    ${classes.line_both}
+                    ${classes.mobile_projects_drop}
+                  `}
+                  onClick={handleProjectsMenuForMobile}
+                  onTouchStart={handleProjectsMenuForMobile}
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Header/index.tsx]: Do not use onClick on div's, use button instead

File: frontend/src/components/Header/index.tsx [hunk 4]
```diff
@@ -1,8 +1,17 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Header/index.tsx]: remove this

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/components/Header/index.tsx (hunk 2 above)
Comment (maksim-luzin) [reply frontend/src/components/Header/index.tsx]: I replaced.

**Pull request merged**